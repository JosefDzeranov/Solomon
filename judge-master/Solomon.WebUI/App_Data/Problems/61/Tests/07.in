53 97
ammo
ammo
shoot
shoot
ammo
shoot
shoot
shoot
shoot
shoot
ammo
shoot
shoot
shoot
ammo
shoot
shoot
ammo
ammo
shoot
shoot
ammo
shoot
shoot
ammo
shoot
shoot
shoot
shoot
shoot
shoot
ammo
shoot
shoot
ammo
ammo
ammo
shoot
shoot
shoot
shoot
ammo
ammo
shoot
ammo
ammo
shoot
shoot
shoot
ammo
shoot
ammo
ammo
