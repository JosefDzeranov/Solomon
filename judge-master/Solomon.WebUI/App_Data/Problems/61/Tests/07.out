97
194
193
192
289
288
287
286
285
284
381
380
379
378
475
474
473
570
667
666
665
762
761
760
857
856
855
854
853
852
851
948
947
946
1043
1140
1237
1236
1235
1234
1233
1330
1427
1426
1523
1620
1619
1618
1617
1714
1713
1810
1907
